README.md

```diff
@@ -102,9 +102,19 @@ Eg:
 -Dgo.plugin.build.status.gerrit.codeReviewLabel=Verified
 ```
 
+#### Gitlab
+**Setup:**
+- You need to provide `endpoint`, `oauth token` through system property `go.plugin.build.status.gitlab.endpoint`, `go.plugin.build.status.gitlab.oauth`.
+Eg:
+```
+-Dgo.plugin.build.status.gitlab.endpoint=http://gitlab.com
+-Dgo.plugin.build.status.gitlab.oauth=XXXX
+```
+
 ## FAQs
 
 [1]: images/list-plugin.png  "List Plugin"
 [2]: images/configure-plugin.png  "Configure Plugin"
-[3]: images/pipeline-schedule.png  "Pipeline Schedule"
-[4]: images/update-status.png  "On Successful Status Update"
+[3]: images/gitlab-plugin-settings.png  "Configure Gitlab Plugin"
+[4]: images/pipeline-schedule.png  "Pipeline Schedule"
+[5]: images/update-status.png  "On Successful Status Update"
```

Comment:
`5` seems unused in the readme?

Reply:
fixed the order and referenced the image in Gitlab config section